pkg/schema/load/generic.go

```diff
@@ -88,10 +89,38 @@ func buildGenericScuemata(famval cue.Value) (schema.VersionedCueSchema, error) {
 		}
 		major++
 	}
-
+	if lastgvs.actual, err = prepare(lastgvs.actual); err != nil {
+		return nil, err
+	}
 	return first, nil
 }
 
+func prepare(iCUE cue.Value) (cue.Value, error) {
+	iter, err := iCUE.Fields(cue.Definitions(true))
+	if err != nil {
+		return iCUE, err
+	}
+	for iter.Next() {
+		a := iter.Value().Attribute("cuetsy")
```

Comment:
This call may work here, but my advice for now is to avoid `cue.Value.Attribute()` _completely_ in favor of ranging over `cue.Value.Attributes()`. The former is both suboptimal (just picks the first when there are multiple matching attributes) and buggy. Better to iterate over the full set.